doc/nrf/ug_nrf52.rst

```diff
@@ -232,6 +232,10 @@ To perform a FOTA upgrade, complete the following steps:
       The :file:`app_update.bin` file is the file that must be downloaded to the device.
 
 #. Download the new image to a device.
+
+      .. note::
+         When performing FOTA upgrade on a Bluetooth mesh device and if the device's composition data is going to change after the firmware upgrade, unprovision the device before downloading the new image.
```

Comment:
```suggestion
         If the device's composition data is going to change after the firmware upgrade when performing FOTA upgrade on a Bluetooth mesh device, unprovision the device before downloading the new image.
```
It feels a bit easier to read to me, just by swapping it around. See if you agree :)

Reply:
Looking at the documentation preview is looks a little bit strange that we start the "Download the new image to a device." with a note and end with a note. The text in the middle disappears somewhat. Could me merge the two notes into 1 note with two sections or something. This is mostly cosmetic.